### PR TITLE
feat: [FFM-10189]: Re-introduce bucket by

### PR DESCRIPTION
**Pull request by knagurski**
This PR re-introduces the Bucket By field to places that display the percentage rollout component.

### CODE REVIEW

**Review by Kira-Jennings — DISMISSED** [review]

**Review by laura-mannus — APPROVED** [review]

**Review by laura-mannus — APPROVED** [review]

File: src/modules/75-cf/components/PipelineSteps/FlagConfigurationStep/FlagChangesV2/subSections/ServePercentageRolloutToTargetGroup/__tests__/BucketByField.test.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    expect(await screen.findByPlaceholderText('cf.percentageRollout.bucketBy.placeholder')).toBeInTheDocument()
+
+    await userEvent.click(await screen.findByRole('button'))
+    await userEvent.click(await screen.findByText('Runtime input'))
+
+    expect(screen.queryByPlaceholderText('cf.percentageRollout.bucketBy.placeholder')).not.toBeInTheDocument()
+    expect(await screen.findByPlaceholderText(RUNTIME_INPUT_VALUE)).toBeInTheDocument()
+  })
+
+  test('it should fetch target attributes when the env, project, org and account IDs are available', async () => {
```
Comment (crystal-fong) [inline src/modules/75-cf/components/PipelineSteps/FlagConfigurationStep/FlagChangesV2/subSections/ServePercentageRolloutToTargetGroup/__tests__/BucketByField.test.tsx:59]: don't have to address right now but could we also assert that if either env/project/org/account is missing, then it will default target attributes to be an empty array?

Comment (knagurski) [reply src/modules/75-cf/components/PipelineSteps/FlagConfigurationStep/FlagChangesV2/subSections/ServePercentageRolloutToTargetGroup/__tests__/BucketByField.test.tsx:59]: I had thought about that. I didn't add a test here because the drop down won't be empty, it'll have the 2 defaults. I suppose I could assert that the hook was called with `lazy: true` to prove it didn't try to do the call 🤔 Either that or look at the menu itself and could the options in it

File: src/modules/75-cf/strings/strings.en.yaml [hunk 2]
```diff
@@ -2227,5 +2228,10 @@ cfTrialHomePage:
     startBtn:
       description: Start 14 day FF Enterprise trial
 percentageRollout:
-  toTargetGroup: 'To Target Group:'
+  toTargetGroup: 'To target group'
```
Comment (crystal-fong) [inline src/modules/75-cf/strings/strings.en.yaml:2231]: nbd but these quotes aren't needed

**Review by crystal-fong — APPROVED** [review]

File: src/modules/75-cf/components/PipelineSteps/FlagConfigurationStep/FlagChangesV2/subSections/ServePercentageRolloutToTargetGroup/__tests__/BucketByField.test.tsx (hunk 1 above)
Comment (crystal-fong) [reply src/modules/75-cf/components/PipelineSteps/FlagConfigurationStep/FlagChangesV2/subSections/ServePercentageRolloutToTargetGroup/__tests__/BucketByField.test.tsx:59]: ye, pretty cool lazy trick as well

File: src/modules/75-cf/strings/strings.en.yaml (hunk 2 above)
Comment (knagurski) [reply src/modules/75-cf/strings/strings.en.yaml:2231]: True. Needed with the `:` in there, but not without
